Gems/Atom/RHI/DX12/Code/Include/Atom/RHI.Reflect/DX12/PlatformLimitsDescriptor.h

```diff
@@ -64,12 +64,18 @@ namespace AZ
             //! int array: Max count for descriptors 
             AZStd::unordered_map<AZStd::string, AZStd::array<uint32_t, NumHeapFlags>> m_descriptorHeapLimits;
 
-            // Number of max static handles for shader visible srv/uav/cbv views
+            //! Number of max static handles for shader visible srv/uav/cbv views
             uint32_t m_numShaderVisibleCbvSrvUavStaticHandles = 2000;
 
-            //Bool to indicate allowing compaction of shader visible srv/uav/cbv heap in case of fragmentation
+            //! Bool to indicate allowing compaction of shader visible srv/uav/cbv heap in case of fragmentation
             bool m_allowDescriptorHeapCompaction = false;
 
+            //! Denote portion of the shader-visible descriptor heap used to maintain static handles.
+            //! NOTE: dynamic descriptors are needed to allocate per-frame descriptor tables for resources that are
+            //! not bound via bindless, so this number should reflect that. If the majority of resources correctly
+            //! leverage the bindless mechanism, this number can be higher (e.g. [0.8f, 0.9f]).
+            float m_staticDescriptorRatio = 0.5f;
```

Comment:
I would set this to 0.1 as a start and once terrain/RT have switched over we can bump this up to 0.9. If we check this in we are essentially cutting the available handles by half for the direct binding model used within Terrain/RT.

Reply:
Well in this commit, every view is associated with a static descriptor also so that would have to change also (but I'm inclined to say just giving everything a static descriptor is fine to start). RT would need to switch binding schemes before this could get merged in anyways, and at that point I think 500000 dynamic descriptors would be plenty